src/Authenticators/TOTPAuthenticator.php

```diff
@@ -26,47 +28,54 @@ class TOTPAuthenticator extends BootstrapMFAAuthenticator
      * a specific TOTP authenticator
      */
     private static $algorithm = 'sha1';
-
-
+    
     /**
-     * @param string $link
-     * @return \SilverStripe\Security\MemberAuthenticator\LoginHandler|static
+     * Get configured algorithm for TOTP Authenticator
+     *
+     * Must be one of: "sha1", "sha256", "sha512"
+     * If not specified or invalid, default to "sha1"
+     * @return string
      */
-    public function getLoginHandler($link)
+    public static function get_algorithm()
```

Comment:
It's a bit murky at the moment, but we are aiming for full PSR-2 compatibility in future, so `getAlgorithm()` might be a better name for standards future proofing

Reply:
Alternatively we could use a service pattern e.g. `TOTPAuthenticator::singleton()->getAlgorithm()`

Reply:
I simply didn't touch it from the original, happy to change

Reply:
Ok didn't realise (have since) - let's do it separately